scripts/install.sh

```
@@ -40,6 +40,34 @@ warn() { echo -e "${YELLOW}[WARN]${NC} $1"; }
 error() { echo -e "${RED}[ERROR]${NC} $1" >&2; }
 success() { echo -e "${GREEN}[OK]${NC} $1"; }
 
+write_install_manifest() {
+  local binary_path="$1"
+  local installed_version="$2"
+  local archon_home="${ARCHON_HOME:-$HOME/.archon}"
+  local manifest_path temp_path escaped_binary escaped_version installed_at
+
+  case "$archon_home" in
+    "~") archon_home="$HOME" ;;
+    "~/"*) archon_home="$HOME/${archon_home#\~/}" ;;
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)